setup.py

```diff
@@ -6,28 +6,46 @@
 
 from setuptools import find_packages, setup
 
-version = '2.2a0'
 
-install_requires = [
+# Package metadata.
+
+name = "django-google-spanner"
+description = "Bridge to enable using Django with Spanner."
+version = "2.2a0"
```

Comment:
I assume release please will replace this version since this doesn't look like the version number it's proposing in #453

Reply:
That wouldn't be ideal. The README explains the typical versioning of database backends: "Use the version of django-spanner that corresponds to your version of Django. For example, django-spanner 2.2.x works with Django 2.2.y. (This is the only supported version at this time.)"

Reply:
@timgraham thanks for that info. But see my [comment](https://github.com/googleapis/python-spanner-django/pull/454#issuecomment-631191745). There are lots of existing django packages that are published by Google and they all consistently use the versioning scheme used by release-please.

Reply:
Yes, sem-var versioning is typical for other Django libraries that often support multiple versions of Django. Database backends are different. The best practice is to create separate release branches for each version of Django that's supported.

Reply:
I spoke to @larkee and they explained that the manual releasing won't be so painful.

> re: versions
> 
> I would prefer semver for consistency with other packages. It also makes it easier for folks to pin on this library (e.g., `google-django-spanner>=0.1.0,<1.0.0` 

@timgraham do you have any thoughts on the issue of pinning releases when you have non-semver versioning?

Reply:
If using Django 2.2.x, use `django-google-spanner>=2.2,<3.0`. I think `django-google-spanner==2.2.*` would be equivalent.

Reply:
In that case, @timgraham or @odeke-em could you update the README to explain why the versioning is the way it is? Please include links to the Django README as a reference.

Reply:
What I quoted is from the README in this repo.